client/cli/src/config.rs

```diff
@@ -405,6 +405,13 @@ pub trait CliConfiguration: Sized {
 		Ok(true)
 	}
 
+	/// Get the format for the informant
+	///
+	/// By default this enables colour and does not use a prefix.
+	fn informant_output_format(&self) -> Result<sc_informant::OutputFormat> {
```

Comment:
@cecton this does not makes any sense and `sc_informant` should also not be a dependency of `sc-cli`.

It was also not used anymore by `sc-cli`.

Why should an option that is not controlled by the CLI, be part of the `CliConfiguration`?

Reply:
CliConfiguration has a bad name but it is the trait that transforms anything into a sc-service::Configuration

Reply:
Ok no that changed during the development and we added the shared_params and other params so I guess CliConfiguration is specialized for CLI.

Reply:
So you can drop it I guess @expenses

Reply:
Ok :+1: